src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorOverloadResolution.cs

```diff
@@ -699,7 +699,7 @@ private void GetAllBuiltInOperators(BinaryOperatorKind kind, BoundExpression lef
                 GetPointerOperators(kind, left, right, operators);
             }
 
-            CandidateOperators(operators, left, right, results, ref useSiteDiagnostics);
+            CandidateOperators(operators, left, right, results, ref useSiteDiagnostics, skipNativeIntegerOperators: !left.Type.IsNativeIntegerOrNullableNativeIntegerType() && !right.Type.IsNativeIntegerOrNullableNativeIntegerType());
```

Comment:
>operators [](start = 31, length = 9)

Instead of filtering candidates from ```operators``` can we instruct GetSimpleBuiltInOperators to not put them there in the first place?